process_pr.py

```diff
@@ -494,7 +495,9 @@ def check_type_labels(first_line, extra_labels):
                 obj_labels = rem_labels if rem_lab else ex_labels
                 if lab_type not in obj_labels:
                     obj_labels[lab_type] = []
-                if (len(TYPE_COMMANDS[lab]) > 3) and TYPE_COMMANDS[lab][3]:
+                if (len(TYPE_COMMANDS[lab]) > 4) and TYPE_COMMANDS[lab][4] == "state":
+                    state_labels[lab_type].append(type_cmd)
```

Comment:
it should be `state_labels[lab].append(lab_type)` where `lab`  will be `<arch>-cms-docker` and  `lab_type` will be the  type comemnt value e.g. `<arch>-runtime-building`

Reply:
@aandvalenzuela , I editted the cooment above . This should be `state_labels[lab]=lab_type`